Gemfile.lock

```diff
@@ -164,6 +170,10 @@ PLATFORMS
   ruby
 
 DEPENDENCIES
+<<<<<<< HEAD
+=======
+  bootstrap-sass (~> 3.3.5)
+>>>>>>> Landing page
```

Comment:
I think this ended up here by accident due to an unresolved conflict, please remove it.